File: official/docs/go/shipments/retrieve_shipment_list.go
package example

import (
	"fmt"
	"os"

	"github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
	client := easypost.New(apiKey)

	shipments, err := client.ListShipments(
		&easypost.ListShipmentsOptions{
			PageSize:      5,
			StartDateTime: "2016-01-02T08:50:00Z",
		},
	)

	fmt.Println(shipments)
}

File: official/docs/php/shipments/RetrieveShipmentList.php
<?php

\EasyPost\EasyPost::setApiKey("EASYPOST_API_KEY");

$shipments = \EasyPost\Shipment::all([
    "page_size" => 5,
    "start_datetime" => "2016-01-02T08:50:00Z"
]);

echo $shipments;
